README.md

```diff
@@ -194,18 +226,18 @@ desired points in the life of a virtual node.
 
 #### Overview
 
-| Name        | Triggered when                                     | Arguments to callback   |
-| ----------- | --------------                                     | ----------------------- |
-| `pre`       | the patch process begins                           | none                    |
-| `init`      | a vnode has been added                             | `vnode`                 |
-| `create`    | a DOM element has been created based on a vnode    | `emptyVnode, vnode`     |
-| `insert`    | an element has been inserted into the DOM          | `vnode`                 |
-| `prepatch`  | an element is about to be patched                  | `oldVnode, vnode`       |
-| `update`    | an element is being updated                        | `oldVnode, vnode`       |
-| `postpatch` | an element has been patched                        | `oldVnode, vnode`       |
-| `destroy`   | an element is directly or indirectly being removed | `vnode`                 |
-| `remove`    | an element is directly being removed from the DOM  | `vnode, removeCallback` |
-| `post`      | the patch process is done                          | none                    |
+| Name | Triggered when | Arguments to callback |
+| ---- | -------------- | --------------------- |
+| `pre` | the patch process begins | none |
+| `init` | a vnode has been added | `vnode` |
+| `create` | a DOM element has been created based on a vnode | `emptyVnode, vnode` |
+| `insert` | an element has been inserted into the DOM | `vnode` |
+| `prepatch` | an element is about to be patched | `oldVnode, vnode` |
+| `update` | an element is being updated | `oldVnode, vnode` |
+| `postpatch` | an element has been patched | `oldVnode, vnode` |
+| `destroy` | an element is directly or indirectly being removed | `vnode` |
+| `remove` | an element is directly being removed from the DOM | `vnode, removeCallback` |
+| `post` | the patch process is done | none |
```

Comment:
Since remark re-formats, it is either re-formatting this to be compact or re-formatting the [maintenance policy tables](https://github.com/snabbdom/snabbdom/blob/master/README.md#maintenance-policy) as padded, which kinda sucks, I feel.